src/main.js

```diff
@@ -94,25 +211,24 @@ async function summarize(coverageFile) {
     },
     stderr: (data) => {
       output += data.toString();
-    }
+    },
   };
 
-  await exec.exec('lcov', [
-    '--summary',
-    coverageFile,
-  ], options);
+  await exec.exec(
+    "lcov",
+    ["--summary", coverageFile, "--rc", "lcov_branch_coverage=1"],
+    options
+  );
 
-  const lines = output
-    .trim()
-    .split(/\r?\n/);
+  const lines = output.trim().split(/\r?\n/);
 
   lines.shift(); // Removes "Reading tracefile..."
 
-  return lines.join('\n');
+  return lines.join("\n");
```

Comment:
I do like formatters, but in this case, if we're trying to stay close to upstream, I'd rather have inconsistent formatting but a smaller diff. (Or if upstream uses a formatter, make sure we use the same settings as them.)

Reply:
Upstream doesn't have a formatter; it's the default that my editor uses. I switched to using a specific formatter instead, so that it'll be consistent going forwards.